app/src/main/java/com/sunasterisk/fooddaily/data/source/remote/response/SearchResponseHandler.kt

```diff
@@ -0,0 +1,50 @@
+package com.sunasterisk.fooddaily.data.source.remote.response
+
+import com.sunasterisk.fooddaily.data.model.FoodDetail
+import org.json.JSONObject
+import java.net.HttpURLConnection
+import java.net.URL
+import java.util.*
+import kotlin.collections.ArrayList
+
+class SearchResponseHandler: DataResponseHandler {
```

Comment:
Chỉ cần 1 thằng Handler là đủ 
```suggestion
object SearchResponseHandler: DataResponseHandler {
```

Reply:
chỗ này mình chưa hiểu :\
Huy cho mình xin keyword để tìm hiểu về cách dùng này với

Reply:
keyword là object kotlin